============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-4.6.2, py-1.8.0, pluggy-0.12.0 -- /home/deilson/codenation/data-science-2/m5/bin/python
cachedir: .pytest_cache
rootdir: /home/deilson/codenation/data-science-2
collecting ... collected 7 items

test_submit.py::test_q1 PASSED                                           [ 14%]
test_submit.py::test_q2 PASSED                                           [ 28%]
test_submit.py::test_q3 PASSED                                           [ 42%]
test_submit.py::test_q4 PASSED                                           [ 57%]
test_submit.py::test_q5 PASSED                                           [ 71%]
test_submit.py::test_q6 PASSED                                           [ 85%]
test_submit.py::test_q7 PASSED                                           [100%]

=============================== warnings summary ===============================
m5/lib/python3.7/site-packages/statsmodels/tools/_testing.py:19
  /home/deilson/codenation/data-science-2/m5/lib/python3.7/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
    import pandas.util.testing as tm

main.py:121
  /home/deilson/codenation/data-science-2/main.py:121: UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
    plt.show()

-- Docs: https://docs.pytest.org/en/latest/warnings.html
---- generated xml file: /home/deilson/codenation/data-science-2/output.xml ----
===================== 7 passed, 2 warnings in 0.86 seconds =====================
